cfgov/cfgov/settings/base.py

```diff
@@ -386,6 +386,7 @@
 AWS_S3_FILE_OVERWRITE = False
 AWS_S3_SECURE_URLS = True  # True = use https; False = use http
 AWS_STORAGE_BUCKET_NAME = os.environ.get('AWS_STORAGE_BUCKET_NAME')
+AWS_DEFAULT_ACL = None  # Default to using the ACL of the bucket
```

Comment:
I see. No, not intentional. I took the documentation's recommendation without enough thought as to what we have those values set to.

Since this is complex, I created a separate ticket for it. I'll revert this change for this PR.